scripts/manage-members.js

```diff
@@ -0,0 +1,177 @@
+'use strict'
+
+const P = require('effd')
+const request = require('request')
+const debug = require('debug')('manage-members')
+const logger = require('../lib/logger.js')
+
+var login
+const github = require('../lib/github-client.js')
+// store/cache info about the authenticated user
+github.users.get({}).then((user) => {
```

Comment:
Sure- it could probably be lazily loaded. But either way- those other tests should only load the scripts they are testing.